diff --git a/internal/pkg/remote/endpoint/service.go b/internal/pkg/remote/endpoint/service.go
index 21404d87b..031755fc9 100644
--- a/internal/pkg/remote/endpoint/service.go
+++ b/internal/pkg/remote/endpoint/service.go
@@ -121,7 +121,7 @@ func (ep *Config) GetAllServices() (map[string][]Service, error) {
 		Timeout: defaultTimeout,
 	}
 
-	url := "https://" + ep.URI + "/assets/config/config.prod.json"
+	url := "http://" + ep.URI + "/assets/config/config.prod.json"
 
 	req, err := http.NewRequest(http.MethodGet, url, nil)
 	if err != nil {
